### PR TITLE
Address lint errors

### PR DESCRIPTION
**Pull request by achew22**

### CODE REVIEW

**Comment by achew22** [comment]
@laurentlb it looks like it is not possible to fix all the linter errors in the repo. See https://github.com/bazelbuild/buildtools/issues/865 for a more complete explanation of this. Can you advise on how to proceed?

File: rules/common_settings.bzl [hunk 1]
```diff
@@ -12,6 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# buildifier: disable=skylark-docstring
+# buildifier: disable=skylark-comment
```
Comment (vladmos) [inline rules/common_settings.bzl:16]: These comments are not required anymore, the latest version of buildifer shouldn't warn about the usage of this url anymore.

**Comment by vladmos** [comment]
I see just one warning in the CI, it should disappear if you rerun the tests (the CI automatically uses the latest version of buildifier released). Also the disabling comments aren't required anymore.

File: rules/common_settings.bzl (hunk 1 above)
Comment (achew22) [reply rules/common_settings.bzl]: Done.

**Comment by achew22** [comment]
Looks like that fixed it. Thanks for addressing the linter issues @vladmos

**Pull request merged**